onnxmltools/convert/sklearn/operator_converters/NaiveBayes.py

```diff
@@ -0,0 +1,210 @@
+    #  onnx_proto.TensorProto.INT64 -> CAST --------> output[M,]
+    #
+    # Bernoulli NB
+    # Equation:
+    #   y = argmax (class_log_prior + Σ neg_prob - X . neg_prob)
+    #   neg_prob = log( 1 - e ^ feature_log_prob)
+    #
+    #   Graph:
+    #
+    #  feature_log_prob.T[N, C] -> EXP -> exp_result[N, C] 
+    #                                      |
+    #                                      V
+    #                         constant -> SUB -> sub_result[N, C] -> LOG -> neg_prob[N, C]
+    #                                                                        |
+    #                                                                        V
+    #  ----------------- sum_neg_prob[1, C] <------------------------ REDUCE_SUM
+    #  |                     |
+    #  |                     V
+    #  |    input[M, N] -> MATMUL -> inp_neg_prob_prod[M, C] -> CAST <- onnx_proto.TensorProto.FLOAT
+    #  |                                                         |
+    #  |                                                         V
+    #  --------------------------------------> SUB  <- cast_result[M, C]
+    #                                           |
+    #                                           V
+    #  class_log_prior[1, C] -> SUM <- difference_matrix[M, C]
+    #                            |
+    #                            V
+    #                       sum_result[M, C] -> ARGMAX -> argmax_output[M, 1] 
+    #                                                            |
+    #                                                            V
+    #                              classes[C] -------> ARRAYFEATUREEXTRACTOR
+    #                                                            |
+    #                                                            V          (string labels)
+    #                                  array_feature_extractor_result[M, 1] ----------------------------
+    #                                               (int labels) |                                     | 
+    #                                                            V                                     |
+    #      output_shape[1] -> RESHAPE <- cast2_result[M, 1] <- CAST <- onnx_proto.TensorProto.FLOAT    | 
+    #                          |                                                                       |
+    #                          V                                                                       |
+    #                       reshaped_result[M,]             --------------------------------------RESHAPE
+    #                                   |                   |
+    #                                   V                   |
+    #  onnx_proto.TensorProto.INT64 -> CAST -> output[M,] <-|
+
+    nb = operator.raw_operator
+    class_log_prior = nb.class_log_prior_.astype('float32').reshape((1, -1))
+    feature_log_prob = nb.feature_log_prob_.T.astype('float32')
+    classes = nb.classes_
+    output_shape = [-1,]
+
```

Comment:
It looks like Flatten fits your need better: [https://aiinfra.visualstudio.com/Lotus/Lotus%20Team/_git/Lotus?path=%2Fonnxruntime%2Ftest%2Fpython%2Fonnxruntime_test_python.py&version=GBmaster&line=147&lineStyle=plain&lineEnd=148&lineStartColumn=1&lineEndColumn=1](https://aiinfra.visualstudio.com/Lotus/Lotus%20Team/_git/Lotus?path=%2Fonnxruntime%2Ftest%2Fpython%2Fonnxruntime_test_python.py&version=GBmaster&line=147&lineStyle=plain&lineEnd=148&lineStartColumn=1&lineEndColumn=1).